core/src/main/scala/com/codacy/analysis/core/clients/CodacyClient.scala

```diff
@@ -84,16 +86,18 @@ class CodacyClient(credentials: Credentials, http: HttpHelper)(implicit context:
 
   private def sendRemoteResultsTo(endpoint: String,
                                   tool: String,
-                                  results: Set[FileResults]): Future[Either[String, Unit]] =
+                                  results: Either[String, Set[FileResults]]): Future[Either[String, Unit]] =
     Future {
       http.post(endpoint, Some(Seq(ToolResults(tool, results)).asJson)) match {
         case Left(error) =>
           logger.error(error)(s"Error posting data to endpoint $endpoint")
           Left(error.message)
         case Right(json) =>
-          logger.info(s"""Success posting batch of ${results.size} files with ${results
-            .map(_.results.size)
-            .sum} results to endpoint "$endpoint" """)
+          results.fold(
+            error => logger.info(s"Success posting analysis error $error"),
+            results => logger.info(s"""Success posting batch of ${results.size} files with 
+                ${results.map(_.results.size).sum} results to endpoint "$endpoint" """))
```

Comment:
Add the "|" at the beginning of the line and ."stripMargin" at the end of the String, so we don't get all of the spacing to the left on the log.